src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -0,0 +1,150 @@
+      rowIndex = 1;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(0);
+      expect(nextCol).to.eql(0);
+    });
+
+    it('should go to one column left cell', () => {
+      evt.key = 'ArrowLeft';
+      rowAndColumnCount.columnCount = 2;
+      colIndex = 1;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(0);
+      expect(nextCol).to.eql(0);
+    });
+
+    it('should go to one column right cell', () => {
+      evt.key = 'ArrowRight';
+      rowAndColumnCount.columnCount = 2;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(0);
+      expect(nextCol).to.eql(1);
+    });
+  });
+
+  describe('getRowAndColumnCount', () => {
+    let resolvedRowCount;
+    let resolvedColumnCount;
+    let resolvedRowElements;
+    const rootElement = {};
+
+    it('should return a rowElements, rowCount, and columnCount', () => {
+      resolvedRowCount = 1;
+      resolvedColumnCount = 1;
+      resolvedRowElements = [{ getElementsByClassName: () => {} }];
+      rootElement.getElementsByClassName = () => resolvedRowElements;
+      const { rowElements, rowCount, columnCount } = getRowAndColumnCount(rootElement);
+      expect(rowCount).to.equal(resolvedRowCount);
+      expect(columnCount).to.equal(resolvedColumnCount);
+      expect(rowElements).to.equal(resolvedRowElements);
+    });
+  });
+
+  describe('handleKeyPress', () => {
+    const sandbox = sinon.createSandbox();
+    let evt;
+    let rowIndex;
+    let colIndex;
+    const rootElement = {};
+
+    beforeEach(() => {
+      evt = {};
+      rowIndex = 0;
+      colIndex = 0;
+      evt.key = 'ArrowDown';
+      evt.target = {};
+      evt.target.blur = () => {};
+      evt.target.setAttribute = () => {};
+      evt.stopPropagation = () => {};
+      evt.preventDefault = () => {};
+      rootElement.getElementsByClassName = () => [
+        { getElementsByClassName: () => [{ focus: () => {}, setAttribute: () => {} }] },
+      ];
+      sandbox.replace(handleKeyPress, 'default', sandbox.spy());
+      sandbox.replace(handleKeyPress, 'preventDefaultBehavior', sandbox.spy());
+      sandbox.replace(handleKeyPress, 'removeCurrentFocus', sandbox.spy());
+      sandbox.replace(handleKeyPress, 'getRowAndColumnCount', sandbox.spy());
+      sandbox.replace(handleKeyPress, 'arrowKeysNavigation', sandbox.spy());
+      sandbox.replace(handleKeyPress, 'moveToNextFocus', sandbox.spy());
+    });
+
+    afterEach(() => {
+      sandbox.verifyAndRestore();
+    });
+
+    it('should stay current focus and focus the next cell', () => {
+      handleKeyPress.default(evt, rootElement, rowIndex, colIndex);
+      expect(handleKeyPress.default).to.have.been.calledOnce;
+      expect(handleKeyPress.preventDefaultBehavior).to.have.been.calledOnce;
```

Comment:
I got AssertionError: expected spy to have been called exactly once, but it was called 0 times.